1.10/cli/command-reference/dcos-cluster/dcos-cluster-attach.md

```diff
@@ -4,7 +4,7 @@ menu_order: 2
 ---
 
 # Description
-Connect to DC/OS clusters.
+Attach to a connected DC/OS cluster. When you run the [dcos cluster setup](/docs/1.10/cli/command-reference/dcos-cluster/dcos-cluster-setup) command, the cluster is automatically attached.
```

Comment:
I don't know if it's necessary because of context, but what to you think of "Attach the CLI to a DC/OS cluster"?